docs/language/learn-ql/writing-queries/debugging-queries.rst

```diff
@@ -81,6 +81,48 @@ That is, you should define a *base case* that allows the predicate to *bottom ou
    The query optimizer has special data structures for dealing with `transitive closures <https://help.semmle.com/QL/ql-handbook/recursion.html#transitive-closures>`__.
    If possible, use a transitive closure over a simple recursive predicate, as it is likely to be computed faster.
 
+Folding predicates
```

Comment:
```suggestion
Fold predicates
```

This would match the other titles, but I admit it sounds a bit odd.

Reply:
Sounds ok to me. Thanks!